config/config.toml

```diff
@@ -40,6 +40,13 @@ enableProcessor = true
 enableRouter = true
 enableStats = true
 
+[Http]
+ReadTimeOutInSec = 1
+ReadHeaderTimeoutInSec = 2
+WriteTimeOutInSec = 5
+IdleTimeoutInSec = 30
```

Comment:
We should keep really high value for idle timeout, as I have set keepalive time in nginx to 600 sec. In our server we should set it an even higher value. Let's keep this like 720 or something. Otherwise, what will happen is server will terminate the keep alive connection, whereas nginx/elb will forward request assuming connection is alive.